libraries/AP_MSP/msp_sbuf.cpp

```diff
@@ -0,0 +1,48 @@
+#include "msp.h"
+#include "msp_sbuf.h"
+#include <string.h>
+
+#if HAL_MSP_ENABLED
+
+uint8_t* MSP::sbuf_ptr(sbuf_t *buf)
```

Comment:
I did a C++ version of sbuf, but now appear to have deleted it. The betaflight version is kind of sucky.

Reply:
I saw it, I think issue is more related to bounds checking